modules/gapi/include/opencv2/gapi/infer.hpp

```diff
@@ -133,31 +133,35 @@ struct InOutInfo
  * @{
```

Comment:
> `@{`

Why do we have this here?

/cc @dmatveev

Reply:
Likely a copy-paste issue, cc: @TolyaTalamanov